libpod/runtime_ctr.go

```diff
@@ -905,3 +907,32 @@ func (r *Runtime) PruneContainers(filterFuncs []ContainerFilter) (map[string]int
 	}
 	return prunedContainers, pruneErrors, nil
 }
+
+// StorageContainers returns a list of containers from containers/storage that
+// are not currently known to Podman.  The list of containers passed in are all
+// of the containers known to Podman from the GetContainers() function.
+func (r *Runtime) StorageContainers() ([]storage.Container, error) {
+	var retCtrs []storage.Container
+
+	// We only have a store when doing all, storage or size, so skip this otherwise.
+	if r.store == nil {
+		return retCtrs, nil
```

Comment:
This should probably be an error

Reply:
Are you sure?  All of the other checks ignore this issue and just return success?

Reply:
Having no store is something that we deliberately did ourselves - we should never be calling this in a case where the store is not set up, so returning an error seems correct